src/WebJobs.Extensions.DurableTask/EntityScheduler/TypedInvocationExtensions.cs

```diff
@@ -114,8 +110,8 @@ internal static MethodInfo FindMethodForContext<T>(IDurableEntityContext context
                 return method;
             }
 
-            var entityInterface = interfaces[0];
-            return entityInterface.GetMethod(context.OperationName, bindingFlags);
+            var methods = interfaces.SelectMany(i => i.GetMethods(bindingFlags));
+            return methods.FirstOrDefault(m => m.Name.Equals(context.OperationName));
```

Comment:
Previously this was throwing an `AmbiguousMatchException` when there are multiple matches for the name. It may not be important to do the same check across all interfaces but we could at least do it within each interface, e.g. something like
```
interfaces.Select(i => i.GetMethod(context.OperationName, bindingFlags)).FirstOrDefault(m => m != null);

```

Reply:
fair enough, it does make sense. I was a bit restless tbh by specifying a `BindingFlag.IgnoreCase` without getting the method itself by name.